integration-tests/rpc-nodes.spec.ts

```diff
@@ -1,10 +1,11 @@
 import { CONFIGS } from './config';
-import { DefaultContractType } from "@taquito/taquito";
-import { RpcClientCache, RpcClient, RPCRunViewParam, RPCRunScriptViewParam, PendingOperations, PvmKind } from '@taquito/rpc';
+import { DefaultContractType, Protocols } from "@taquito/taquito";
+import { RpcClientCache, RpcClient, RPCRunViewParam, RPCRunScriptViewParam, PendingOperationsV1, PendingOperationsV2, PvmKind } from '@taquito/rpc';
```

Comment:
what are the differences between V1 and V2?

Reply:
PendingOperationsV1 shows {applied: { kind: 'endorsement' } }
PendingOperationsV2 shows {validated: { kind: 'attestation' } }